### PR TITLE
14.0 mig account invoice analytic search

### PR DESCRIPTION
**Pull request by joshburkholder**
Migrate the `account_invoice_analytic_search` module from 12 to 14

### CODE REVIEW

**Review by JasminSForgeFlow — APPROVED** [review]
Functional, LGTM!

File: account_invoice_analytic_search/__manifest__.py [hunk 1]
```diff
@@ -4,8 +4,8 @@
 {
     "name": "Invoice Analytic Search",
     "summary": "Search invoices by analytic account or by project manager",
-    "version": "12.0.1.0.0",
-    "author": "Project Expert Team," "Eficent," "Odoo Community Association (OCA)",
+    "version": "14.0.1.0.0",
+    "author": "Project Expert Team, Eficent, Odoo Community Association (OCA)",
```
Comment (AaronHForgeFlow) [inline account_invoice_analytic_search/__manifest__.py:8]: Can you please change Eficent by ForgeFlow? Thanks

**Review by AaronHForgeFlow — COMMENTED** [review]
LGTM. @joshburkholder  do you mind to rebase your branch? there are several months since the PR was created and it is safer if we check CI before merging. Also would you mind to change author as suggested?

**Pull request closed without merging**